src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.Packages.cs

```diff
@@ -48,9 +51,8 @@ public OutdatedPackageException(string message, PackageInfo package, NuGetVersio
 
                 var package = PackageInfo.ParsePackageFileName(fileName);
 
-                if (package.IsRoslynToolsetCompiler)
+                if (packagesToBeIgnored.Any(p => p == package.PackageName))
```

Comment:
I think we would want to keep the separate check for RoslynToolsetCompiler. I know the package is the default value. But, if Roslyn wanted to adopt this setting in the future, we would have to remember to include the VS.Tools.Roslyn package name.

Reply:
changed